pkg/capture/capture.go

```diff
@@ -470,13 +426,13 @@ func (c *Capture) capturePacket(pkt capture.Packet) (err error) {
 		return fmt.Errorf("capture error: %w", err)
 	}
 
-	// Populate the GPPacket
+	// Populate the types.GPPacket
```

Comment:
Similar to above, IMHO I don't think `GPPacket` should be in the global `types`. I wouldn't worry about it though, I'll remove the whole thing as part of #83